applications/qwilt/README.md

```diff
@@ -0,0 +1,87 @@
+
+## **Pre Requisites – Resources Required**
+
+Qwilt Nodes Managed Content Resource Profiles
+
+| **Virtual Platforms**              | **(M) Medium**                                              | **(L) Large**        |
+| ---------------------------------- | ----------------------------------------------------------- | -------------------- |
+| **Service Performance**            |                                                             |                      |
+| Classification (Filtered Port 80)  | NA                                                          | NA                   |
+| Delivery                           | 5  Gbps                                                     | 10  Gbps             |
+| **Required Application Resources** |                                                             |                      |
+| Application                        | Content Delivery                                            |                      |
+| Compute* (vCores)                  | 10                                                          | 16                   |
+| Memory (RAM)                       | 64 GB                                                       | 128 GB               |
+| Storage (System minimum)           | 200  GB                                                     | 600  GB              |
+| Storage (Content maximum)          | 20 TB                                                       | 30 TB                |
+| Storage Performance                | 6  GBps   1600 IOPS                                         | 12  GBps   3200 IOPS |
+| Host OS                            | CentOS 7.x                                                  |                      |
+| Network                            | 10G / 40G / 100G / Virtio  Direct-Attach / SR-IOV / Virtual |                      |
+
+## **Where to Purchase**
+
+Qwilt CDN application image can be obtained from Qwilt's portal. For more information visit [www.qwilt.com](http://www.qwilt.com) or email sales@qwilt.com
+
+
+
+## Pre Requisites - Installing OpenNESS
+
+This document assumes that OpenNESS was installed through [OpenNESS playbooks](https://github.com/open-ness/specs/blob/master/doc/getting-started/network-edge/controller-edge-node-setup.md). Certain configurations had been changed to support VM deployment on OpenNESS for network edge following [openness-network-edge-vm-support.md](https://github.com/open-ness/specs/blob/master/doc/applications-onboard/openness-network-edge-vm-support.md).
+
+
+
+## Installing Qwilt CDN application using helm
+
+Run the following commands to deploy Qwilt CDN application through helm:
+
+`helm install qwiltcdn ./helm/qwiltcdn`
+
+ Sample Output would look like:
+
+> `helm install qwiltcdn /root/openness-20.06/helm/qwiltcdn`
```

Comment:
Editorial: snippets of shell inputs & outputs are better bound with ` ```shell ``` ` block.

Reply:
Done, please review